src/modules/entities/normalized-user-types.ts

```diff
@@ -0,0 +1,21 @@
+import {Skill, UserName} from "../users/user-types";
+
+export type NormSkill = Skill;
+
+export interface NormUser extends UserName {
+    id: string;
+    regnalNumber: number;
+    skills: string[];
+}
```

Comment:
Tohle není vyloženě špatně, ale raději bych použil `User`. Je tam problém s jiným typem pro `skills`, ale to jde vyřešit přes `Omit`.

```suggestion
export interface NormUser extends Omit<User, 'skills'> {
    skills: string[];
}
```